Networking/Networking/Model/SitePost.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+
+/// Represents a SitePost Entity.
+///
+public struct SitePost: Codable {
```

Comment:
nit: just wondering if there were any reasons you didn't pick `Post` for the name?

Reply:
Good question. The main reason is that the Post is referred to the Site, but I agree with you, it makes more sense to call it `Post` since we don't need any reference here to Site.